init.sh

```diff
@@ -9,6 +9,8 @@ forking=$4
 nodePath=$5
 adapterPath=$6
 compilerPath=$7
+testFilter=$8
+verboseLogging=$9
```

Comment:
This breaks the command example in the README. But those can be entirely optional right? I.e. you assign a default value if those were not given.

Reply:
Great catch. Ill add some default values for the command line vars so the README examples still work.